frontend/src/components/Forms/FormLayoutCenter/index.js

```diff
@@ -0,0 +1,13 @@
+import { PropTypes } from "prop-types";
+
+import styles from "./index.module.css";
+
+const FormLayoutCenter = ({ children }) => {
+  return <div className={styles.centerLayout}>{children}</div>;
+};
+
+FormLayoutCenter.propTypes = {
+  children: PropTypes.element.isRequired,
+};
+
+export default FormLayoutCenter;
```

Comment:
I don't think you need this component at all.

You can achieve the same by adding `justify-content: center;` to your `otherInterestComponent` class in `SignupPage`

Reply:
You're right, much better! ✨🙏